app/components/meeting-form.hbs

```diff
@@ -4,7 +4,7 @@
       <AuLink @linkRoute="inbox.meetings" @skin="secondary" @icon="arrow-left" @iconAlignment="left">
         {{t "meetingForm.backButton"}}
       </AuLink>
-      <span class="au-c-app-chrome__entity">{{get (await this.currentSession.group) 'classificatie.label'}} {{get (await this.currentSession.group) 'naam'}}</span>
+      <span class="au-c-app-chrome__entity">{{this.currentSession.groupclassificatie.label}} {{this.currentSession.group.naam}}</span>
```

Comment:
missing a `.` here, should be `this.currentSession.group.classificatie.label`

Reply:
Woops, will fix!

Reply:
Should be good now.